plugins/headlamp-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kinvolk/headlamp-plugin",
-  "version": "0.9.2",
+  "version": "0.10.0",
```

Comment:
Can you please leave the version as is? It can be incremented in a separate commit later.